src/simulation/m_start_up.fpp

```diff
@@ -1397,6 +1428,9 @@ contains
     end subroutine s_save_data
 
     subroutine s_initialize_modules
+
+        integer :: i,j,k,l
```

Comment:
Are these used anywhere?

Reply:
These variables were used for debugging purposes, and I am now removing them.

Reply:
to be removed still?

Reply:
Fixed